File: CPP/Program/Anggota.cpp
#include <iostream>
#include <string>

using namespace std;

class Anggota
{
    private:
        string id;
        string nama;
        string bidang;
        string partai;

    public:
        Anggota()
        {
            this->id = "";
            this->nama = "";
            this->bidang = "";
            this->partai = "";
            
        }

        Anggota(string id, string nama, string bidang, string partai)
        {
            this->id = id;
            this->nama = nama;
            this->bidang = bidang;
            this->partai = partai;
        }

        string getId(){
            return this->id;
        }
        string getNama(){
            return this->nama;
        }
        string getBidang(){
            return this->bidang;
        }
        string getPartai(){
            return this->partai;
        }

        void setId(string id){
            this->id = id;
        }
        void setNama(string nama){
            this->nama = nama;
        }
        void setBidang(string bidang){
            this->bidang = bidang;
        }
        void setPartai(string partai){
            this->partai = partai;
        }

        ~Anggota()
        {}
};

File: Python/Program/Anggota.py
class Anggota:
    # Private Attributes
    __id = ""
    __nama = ""
    __bidang = ""
    __partai = ""
    
    # Constructor
    def __init__(self, id = "", nama = "", bidang = "", partai = ""):
        self.__id = id
        self.__nama = nama
        self.__bidang = bidang
        self.__partai = partai
        
    # Getter and Setter
    # Get id
    def get_id(self):
        return self.__id
    
    # Get nama
    def get_nama(self):
        return self.__nama
    
    # Get bidang 
    def get_bidang(self):
        return self.__bidang
    
    # Get partai
    def get_partai(self):
        return self.__partai
    
    # Set id
    def set_id(self, id):
        self.__id = id
        
    # Set nama
    def set_nama(self, nama):
        self.__nama = nama
        
    # Set bidang
    def set_bidang(self, bidang):
        self.__bidang = bidang
        
    # Set partai
    def set_partai(self, partai):
        self.__partai = partai
        

    